app/experimenter/experiments/tests/api/v5/test_mutation.py

```diff
@@ -268,6 +289,52 @@ def test_update_experiment_error(self):
         )
         self.assertEqual(result["status"], 200)
 
+    def test_update_experiment_documentation_links(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create(status=NimbusExperiment.Status.DRAFT)
+        experiment_id = experiment.id
+
+        documentation_links = [
+            {"title": "foo", "link": "https://example.com/bar"},
+            {"title": "baz", "link": "https://example.com/quux"},
+            {"title": "xyzzy", "link": "https://example.com/plotz"},
+        ]
+
+        response = self.query(
+            UPDATE_EXPERIMENT_DOCUMENTATION_LINKS_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "clientMutationId": "randomid",
+                    "documentationLinks": documentation_links,
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+
+        self.assertEqual(response.status_code, 200)
+        content = json.loads(response.content)
+        result = content["data"]["updateExperiment"]
+        self.assertEqual(
+            result["nimbusExperiment"],
+            {
+                "id": experiment.id,
+                "status": experiment.status.upper(),
+                "name": experiment.name,
+                "slug": experiment.slug,
+                "documentationLinks": documentation_links,
```

Comment:
Will this ordering be stable in this test?  Assertions around lists can randomly reorder depending on what phase the moon is in so it's usually better to do a set assertion.

Reply:
Explicitly ordering on ID, so that order should at least be stable